pydocx/export/base.py

```diff
@@ -547,7 +573,5 @@ def export_textbox_content(self, textbox_content):
         return self.yield_nested(textbox_content.children, self.export_node)
 
     def export_markup_compatibility_alternate_content(self, alternate_content):
-        return self.yield_nested(alternate_content.children, self.export_node)
-
-    def export_markup_compatibility_fallback(self, fallback):
-        return self.yield_nested(fallback.children, self.export_node)
+        if self.first_pass:
+            self.alternate_contents.append(alternate_content)
```

Comment:
Kyle is suggesting altering our data model of the XML, not the XML itself, right? That's definitely a cleaner solution than adding more global state (self.alternate_contents) that triggers only on the first pass.